modules/031-local-path-provisioner/images/helper/dir-manager/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"syscall"
+)
+
+var (
+	dirMode     string
+	path        string
+	sizeInBytes string
+	action      string
+)
+
+const (
+	SETUP    = "create"
+	TEARDOWN = "delete"
+)
+
+func init() {
+	flag.StringVar(&path, "p", "", "Absolute path")
+	flag.StringVar(&sizeInBytes, "s", "", "Size in bytes")
+	flag.StringVar(&dirMode, "m", "", "Dir mode")
+	flag.StringVar(&action, "a", "", fmt.Sprintf("Action name. Can be '%s' or '%s'", SETUP, TEARDOWN))
+}
+
+func main() {
+	flag.Parse()
+	if action != SETUP && action != TEARDOWN {
+		fmt.Fprintf(os.Stderr, "Error. Incorrect action: %s", action)
+		os.Exit(1)
+	}
+
+	if path == "" {
+		os.Stderr.WriteString("Error. Path is empty")
```

Comment:
may be also use fmt.Printf ? To make the code the same.

Reply:
optional

Reply:
good point. I've fixed it